src/genie/libs/parser/iosxe/tests/test_show_interface.py

```diff
@@ -536,7 +536,7 @@ class TestShowInterfacesSwitchport(unittest.TestCase):
             'private_vlan': {
             },
             'capture_mode': False,
-            'trunk_vlans': '1,111,130,131,400,405,410,420,430,439-442,450,451,460,',
+            'trunk_vlans': '1,111,130,131,400,405,410,420,430,439-442,450,451,460,470,480,490,500,616,619,700,709-712,720,723-725,760',
```

Comment:
Please run unit test and attach screen shot.

Reply:
![image](https://user-images.githubusercontent.com/19796896/111235916-f3dea780-85f1-11eb-82ff-469dc6af9f59.png)